src/api/entities/Asset/TransferRestrictions/Percentage.ts

```diff
@@ -46,6 +47,11 @@ export class Percentage extends TransferRestrictionBase<TransferRestrictionType.
    */
   public declare removeRestrictions: NoArgsProcedureMethod<BigNumber>;
 
+  /**
+   * Enables investor ownership percentage statistic for the Asset, which is required before creating restrictions
+   */
+  public declare enableStat: ProcedureMethod<Omit<AddPercentStatParams, 'type'>, void>;
```

Comment:
Ideally this would be a NoArgsProcedure, but I haven't figured out how to make that work when Count needs an arg.

Right now this requires the user invoke the function with an empty object, like:
`asset.transferRestrictions.percentage.enableStat({});`

I based this off TransferRestriction, but in that example they both need args.

Reply:
Actually seems like a NoArgsProcedure is working after all. Not sure what was different when I tried it yesterday.